tla-io/src/main/scala/at/forsyte/apalache/io/json/JsonRepresentation.scala

```diff
@@ -3,8 +3,18 @@ package at.forsyte.apalache.io.json
 /**
  * A representation of json. The concrete implementation may depend on external json libraries. Defines a toString
  * method, to be used when performing IO.
+ *
+ * @tparam T
+ *   The class of the value which the JsonRepresentation uses to reprsent JSON
  */
 trait JsonRepresentation {
+
+  /** The type of tused to represent JSON */
+  type Value
```

Comment:
`T` seems to be unused here, why don't you just make `JsonRepresentation[Value]`?

Reply:
Oh that documentation comment should be removed.

I don’t want to make JasonRepresentation parametric now because it’s more work and not needed to achieve the end I need.

When trying to parameterize it the way you suggested in our meeting I hit problems due to the sub typing currently used in the interfaces. This way seems clean and easy: a 2 line change :)

Reply:
Are you worried that this will cause problems some how?

Reply:
No, it just seemed odd to have both `tparam` and `type`. If you want to just keep value I won't block.

Reply:
Yeah, the tparam docs where left over.  Sorry I missed that.